pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -533,6 +531,14 @@ func (vs *VSphere) Instances() (cloudprovider.Instances, bool) {
 }
 
 func getLocalIP() ([]v1.NodeAddress, error) {
+	// hashtable with VMware-allocated OUIs for MAC filtering
```

Comment:
We should leave a link to any public docs in case we need to reference the list of OUIs in the future.

Reply:
As mentioned by @joshrosso and maybe for a future discussion, should this check be removed altogether? Seems like we removed it for the out-of-tree provider so might be worth considering removing the check here as well.

Reply:
@BaluDontu @SandeepPissay can you folks elaborate on this check? looks like it could be removed.

Reply:
resolving this based on @joshrosso comment in the main thread